### PR TITLE
Update document for migrated CloudSuite 3.0.

### PR DESCRIPTION
**Pull request by Xusine1131**
The document is also uploaded to https://hub.docker.com/orgs/cloudsuite3

### CODE REVIEW

File: docs/benchmarks/data-caching.md [hunk 1]
```diff
@@ -138,6 +138,6 @@ and the client on different sockets of the same machine
   [clientdocker]: https://github.com/parsa-epfl/cloudsuite/blob/master/benchmarks/data-caching/client/Dockerfile "Client Dockerfile"
```
Comment (neo-apz) [inline docs/benchmarks/data-caching.md:138]: Can you also change paths like this to point to another branch other than master?
We will create a new branch with whatever we currently have as master as call it `CSv3`, so all these kind of paths should point to that branch.

**Review by neo-apz — CHANGES_REQUESTED** [review]
Also, please update the main README file saying that this is an archived branch and all the old images are moved to this new organization on dockerhub and for the new version of cloudsuite you should check out the main cloudsuite repos on dockerhub.

File: docs/benchmarks/data-caching.md (hunk 1 above)
Comment (Xusine1131) [reply docs/benchmarks/data-caching.md:138]: Let me directly push to a new branch called `CSv3`, alright?

Comment (neo-apz) [reply docs/benchmarks/data-caching.md:138]: No not now.
We want to have these on the master branch and then we will do that.

**Review by neo-apz — APPROVED** [review]

**Pull request merged**